src/main/java/org/sonarsource/sonarlint/ls/CommandManager.java

```diff
@@ -231,7 +231,7 @@ private CodeAction createResolveIssueCodeAction(Diagnostic diagnostic, String ru
     var workspace = workspaceFoldersManager.findFolderForFile(fileUri).orElseThrow(() -> new IllegalStateException("No workspace found"));
     var workspaceUri = workspace.getUri();
     var resolveIssueAction = new CodeAction(String.format(SONARLINT_ACTION_PREFIX + "Resolve issue violating rule '%s' as...", ruleKey));
-    resolveIssueAction.setKind(CodeActionKind.Empty);
+    resolveIssueAction.setKind(CodeActionKind.QuickFix);
```

Comment:
CodeActions with `""` kind were being forwarded to the client but no longer displayed in the UI. Had to make this change in order for them to appear.